nodepool/files/nodepool.yml

```diff
@@ -132,6 +138,8 @@ providers:
     cloud: phobos_nodepool
     boot-timeout: 120
     hostname-format: nodepool-{label.name}-{provider.name}-{node.id}
+    # Ignore the quota given by public cloud and only respect what we provide as max-servers
+    ignore-provider-quota: True
```

Comment:
I think we can skip this setting for Phobos. We have no OnMetal nodes here and when we start to use Ironic it'll be interesting to see how OpenStack is different to public cloud.